src/blueapi/service/main.py

```diff
@@ -392,8 +393,8 @@ def get_state(runner: WorkerDispatcher = Depends(_runner)) -> WorkerState:
     "/worker/state",
     status_code=status.HTTP_202_ACCEPTED,
     responses={
-        status.HTTP_400_BAD_REQUEST: {"detail": "Transition not allowed"},
-        status.HTTP_202_ACCEPTED: {"detail": "Transition requested"},
+        status.HTTP_400_BAD_REQUEST: {},
+        status.HTTP_202_ACCEPTED: {},
```

Comment:
These (and ln318) were raised as problems by the java code gen but aren't directly related to the PR. Should they be split and/or changed to use `description` which I think might work?